tests/Aspire.MongoDB.Driver.Tests/MongoDbContainerFixture.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Components.Common.Tests;
+using MongoDB.Bson;
+using MongoDB.Driver;
+using Testcontainers.MongoDb;
+using Xunit;
+
+namespace Aspire.MongoDB.Driver.Tests;
+
+public sealed class MongoDbContainerFixture : IAsyncLifetime
+{
+    public MongoDbContainer? Container { get; private set; }
+
+    public string GetConnectionString() => Container?.GetConnectionString() ??
+        throw new InvalidOperationException("The test container was not initialized.");
+
+    public async Task InitializeAsync()
+    {
+        if (RequiresDockerTheoryAttribute.IsSupported)
+        {
+            Container = new MongoDbBuilder()
+                .WithImage("mongo:7.0.5")
```

Comment:
I wonder if we should make it so these versions stay in sync between our tests and the AppHost:

https://github.com/dotnet/aspire/blob/ded1eaef1592000b7c99136d1cc9a9085170f07b/src/Aspire.Hosting/MongoDB/MongoDBBuilderExtensions.cs#L31

Reply:
Yes, definitely!

Reply:
Do the versions need to be surfaced in the api too? If it is just for tests, and wherever they are used in the repo right now, then we can use a shared `.cs` file for that.

Reply:
> Do the versions need to be surfaced in the api too?

No, I don't believe so.

> we can use a shared .cs file for that.

That sounds good to me.

cc @mitchdenny

Reply:
I'll cover this in a follow up PR.